### ISSUE TITLE
Only the first commit is correctly analysed

### ISSUE THREAD

**Issue by n-peugnet** [mathieudutour/github-tag-action#15]
Here is an example:

```
=============================================='Analyzing commit: fix(build): remove picocli warnings and fix release workflow
The release type for the commit is patch
Analyzing commit: 
'Merge pull request #41 from Atlaoui/pass-args-to-processes
Pass args to processes
The commit should not trigger a release
Analyzing commit: 
'feat: add --version CLI option + version in MANIFEST
The commit should not trigger a release
Analyzing commit: 
'docs: update README examples and CLI docs
The commit should not trigger a release
Analyzing commit: 
'feat: pass args to processes in MPI and in peersim.
The commit should not trigger a release
Analyzing commit: 
'add args optionnal parameters to CLI
The commit should not trigger a release
[...]
Analyzing commit: 
'fix(build): use checkout action v1 to fetch tags
The commit should not trigger a release
Analyzing commit: 
'fix(build): create annotated tags on release
The commit should not trigger a release
Analysis of 31 commits complete: patch release
git tag -l 0.1.3
```

And the changelog only contains the first commit as well: https://github.com/Atlaoui/ParallelProgramingInterface/releases/tag/0.1.3

Here is the complete build: https://github.com/Atlaoui/ParallelProgramingInterface/runs/711200554?check_suite_focus=true

**Comment by moodysalem** [comment]
same problem:
https://github.com/Uniswap/uniswap-frontend/runs/710230022?check_suite_focus=true

**Comment by mathieudutour** [comment]
could you try the v4.2?

**Comment by moodysalem** [comment]
my expectation was v4 would point at the latest v4, e.g. v4.2, but it seems that is not the case. is there any reason you wouldn't move the v4 tag to point at v4.2?

**Comment by mathieudutour** [comment]
Right, it should be v4.0. But really github actions should support semver. Dealing with all those tags is just annoying

**Comment by mathieudutour** [comment]
I found the bug. The git config on their server is different from mine and I can't figure out how...

Anyhow, should be fixed on v4.3

**Comment by n-peugnet** [comment]
> my expectation was v4 would point at the latest v4, e.g. v4.2, but it seems that is not the case.

That's what I thought too.

With 4.5 everything is working perfectly, thanks !
